src/index.html

```diff
@@ -8,9 +8,43 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <header></header>
+    <main>
+      <div class="search">
+        <form
+          class="search-bar big-bar"
+          data-qa="big"
+        >
+          <input
+            class="search-input big"
+            data-qa="keypress"
+            placeholder= "Try &quot;Los Angeles&quot;"
+          >
+          <img
+            src="images/Search.svg"
+            alt="icon"
+            class="logo big-logo"
+          >
+        </form>
+        <form
+          class="search-bar small-bar"
+          data-qa="small"
+        >
+          <input
+            placeholder="Try &quot;Los Angeles&quot;"
+            class="search-input small"
+          >
```

Comment:
It's better to change naming a little (the similar as you did with other classes. 
As an example
```suggestion
          <input
            class="search-input big-input"
            data-qa="keypress"
            placeholder= "Try &quot;Los Angeles&quot;"
          >
          <img
            src="images/Search.svg"
            alt="icon"
            class="logo big-logo"
          >
        </form>
        <form
          class="search-bar small-bar"
          data-qa="small"
        >
          <input
            placeholder="Try &quot;Los Angeles&quot;"
            class="search-input small-input"
          >
```